webhooks/README.md

```diff
@@ -1,30 +1,30 @@
 # Webhooks
+
 Fidel uses webhooks to notify your application when relevant events happen in your account. This functionality can be used to receive data from events not triggered by direct API requests or to receive the data in a service that is not responsible for making the API request but needs to consume the response.
 
-We will notify your registered webhook URLs as the event happens. For example, when a customer makes a payment with a linked Mastercard on a participating location, a `transaction.auth` event is sent in real-time to the specified webhook URL with the transactional data in the request payload.
+Fidel will notify your registered webhook URLs as the event happens. For example, when a customer makes a payment with a linked Mastercard on a participating location, a `transaction.auth` event is sent in real-time to the specified webhook URL with the transaction object in the request payload.
+---
```

Comment:
let's add a newline before the `---`, otherwise it may be parsed as a heading instead of a paragraph and a `hr`